Lexplorer/Helpers/LinkHelper.cs

```diff
@@ -5,11 +5,11 @@
 
     public class LinkHelper
     {
-        public static MarkupString CreateUserLink(Account? account, string? ignoreId = null, bool includeAccountId = false)
+        public static MarkupString CreateUserLink(AccountModel? account, string? ignoreId = null, bool includeAccountId = false)
```

Comment:
This seems inconsistent to me - why add a Model suffix only to that specific class? Is this because we're also having a page/razor component named "Account"? Wouldn't it be more consistent to rename that to "AccountDetails.razor"?

Reply:
Here you are absolutely right. For me it seemed easier to add a suffix and that suffix should then also be added to other models and this is wrong from my side. Should be more consistent as you stated. Will fix

Reply:
Fixed